test/parallel/test-http-aborted.js

```diff
@@ -35,27 +35,27 @@ const assert = require('assert');
   }));
 }
 
-{
-  // Don't crash if no 'error' handler on server request.
-
-  const server = http.createServer(common.mustCall(function(req, res) {
-    req.on('aborted', common.mustCall(function() {
-      assert.strictEqual(this.aborted, true);
-      server.close();
-    }));
-    assert.strictEqual(req.aborted, false);
-    res.write('hello');
-  }));
-
-  server.listen(0, common.mustCall(() => {
-    const req = http.get({
-      port: server.address().port,
-      headers: { connection: 'keep-alive' }
-    }, common.mustCall((res) => {
-      res.on('aborted', common.mustCall(() => {
-        assert.strictEqual(res.aborted, true);
-      }));
-      req.abort();
-    }));
-  }));
-}
+// {
```

Comment:
The changes in this PR break this behavior, on both client and server. I am not sure about which direction I should go.